beyond-essence/help-center/language-characteristics.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Language characteristics
+background_image: index.jpg
+---
+
+### English (en-GB)
+
+**General**
+
+- Shop for taking screenshots and testing features: https://exquisite-vines.beyondshop.cloud/cockpit/.
+
+**Formatting**
+
+- Currency formatting: Euro 23,50.
+
+**Grammar**
+
+- Use infinitive in the title of a help article. Do not use gerundium or direct speech.
+
+**Terminology**
+
+- Use a language that can be applied to touch devices (e.g. tablets and smartphones). E.g. use "select" or "navigate to ..." instead of "click" or "hover with your mouse over...".
+- Use the path to a view if the navigation takes more than one step: "In the sidebar of the cockpit, navigate to **Products > Product overview**."
+
+
+### German (de-DE)
+
+**General**
+
+- Shop for taking screenshots and testing features: https://holztische.beyondshop.cloud/cockpit.
+
+**Formatting**
+
+- Currency formatting: 23,50 Euro
+
+**Grammar**
+
+- Start the title of a help article with a verb.
+- Use "Du" for addressing the user.
+- Do not start a sentence with "Es".
+
+**Terminology**
+
+- Use a language that can be applied to touch devices (e.g. tablets and smartphones). E.g. use "wählen" instead of "klicken".
+- Use the path to a view if the navigation takes more than one step: "Navigiere in der Seitenleiste des Cockpits zu **Produkte > Produktübersicht**."
```

Comment:
This example will be outdated soon, when we have the new sidebar (also the relating screenshots).